lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/AbstractParser.java

```diff
@@ -110,18 +110,18 @@ public InputSource resolveEntity(String publicId, String systemId) {
   public void
   processFiles(DefaultHandler the_handler,
                Element the_resource) {
-      for (Iterator iter=the_resource.getChildren(FILE, the_handler.getNs().cc_ns()).iterator(); iter.hasNext();) {
-      the_handler.addFile(((Element)iter.next()).getAttributeValue(HREF));
+      for (Iterator iter=the_resource.getChildren(FILE, the_handler.getNs().getNs()).iterator(); iter.hasNext();) {
```

Comment:
I'm using this style here because the jdom javadoc said
"Sequential traversal through the List is best done with a Iterator since the underlying implement of List.size() may not be the most efficient."

I feel like this is a special case? I like the for style better too though.
http://www.jdom.org/docs/apidocs.1.1/org/jdom/Element.html#getChildren()